docs/guides/tanzu-mission-control_data_protection.md

```diff
@@ -5,9 +5,12 @@ Description: |-
 ---
 # Enable Data Protection
 
-The `tanzu-mission-control_enable_data_protection` resource enables users to activate and set up data protection for a Tanzu Kubernetes Cluster.
+The `tanzu-mission-control_enable_data_protection` resource enables users to activate and set up data protection for a Tanzu Kubernetes Cluster/ClusterGroup.
 Once enabled, users can create instant backups or schedule backups for later.
 
+**NOTE :** To resolve cluster and cluster group data protection conflicts use the below command  
```

Comment:
i think we're not supposed to edit files under `docs/` but instead edit the files under `templates/` and then run some tool to generate docs. i had done it before but forgot how. maybe @ramya-bangera can help.

Reply:
also can we add a line asking to run terraform refresh after delete the data protection resource (so the dependent backupschedule reources also get removed from the state)?

Reply:
Added this line [here](https://github.com/vmware/terraform-provider-tanzu-mission-control/blob/asagarwal/handle-tf-refresh/docs/resources/backup_schedule.md#backup-schedule-resource)

Reply:
I see that is about cluster schedule - cg schedule interop. I was talking about dp - schedule interop.

Reply:
also think we need to handle the templates thing otherwise we will lose those changes when docs are generated next.

Reply:
Used templating for doc generation